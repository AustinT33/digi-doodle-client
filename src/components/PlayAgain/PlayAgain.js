import React from 'react';
import Cookies from 'js-cookie';
import './PlayAgain.css';

class PlayAgain extends React.Component {

    handleDeleteData = () => {
        Cookies.remove('digi-doodle-user', { path: `/` });
    }

    render() {
        return (
            <div className="play-again-container">
                <form aria-label="play-again-form" className="play-again-form">
                    <a href="/" role="button" aria-pressed="false" title="play again?" aria-label="play again?" className="play-again-button" onClick={this.handleDeleteData}>Play Again?</a>
                </form>
            </div>
        );
    }
}


export default PlayAgain;