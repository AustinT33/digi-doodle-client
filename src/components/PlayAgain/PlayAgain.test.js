import React from 'react'
import ReactDOM from 'react-dom'
import PlayAgain from './PlayAgain'
import renderer from 'react-test-renderer'
import { BrowserRouter } from 'react-router-dom'

describe(`PlayAgain component`, () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');

        ReactDOM.render(
            <BrowserRouter>
                <PlayAgain />
            </BrowserRouter>, div);

        ReactDOM.unmountComponentAtNode(div)
    })

    it('renders PlayAgain button', () => {
        const playAgain = renderer.create(<PlayAgain />).toJSON();
        expect(playAgain).toMatchSnapshot();
    })


})