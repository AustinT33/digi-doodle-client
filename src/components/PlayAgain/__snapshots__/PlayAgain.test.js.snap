// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayAgain component renders PlayAgain button 1`] = `
<div
  className="play-again-container"
>
  <form
    aria-label="play-again-form"
    className="play-again-form"
  >
    <a
      aria-label="play again?"
      aria-pressed="false"
      className="play-again-button"
      href="/"
      onClick={[Function]}
      role="button"
      title="play again?"
    >
      Play Again?
    </a>
  </form>
</div>
`;
